packages/jest-types/src/TestResult.ts

```diff
@@ -9,7 +9,7 @@ export type Milliseconds = number;
 
 type Status = 'passed' | 'failed' | 'skipped' | 'pending' | 'todo' | 'disabled';
 
-type Callsite = {
+export type Callsite = {
```

Comment:
is this needed anywhere?

Reply:
Yeah it got added, when I was playing with improving the event typings. Removed it :+1: